.github/workflows/composite-actions/upload-artifact/action.yaml

```diff
@@ -0,0 +1,32 @@
+name: "Upload artifact"
```

Comment:
nit: Other job/action names don't have quotes around the name (https://github.com/solo-io/gloo/pull/10743/files#diff-32867e997fa1948dc4c055c9b2a8bc65d9dc2f6959a62bf3ba3f39b2f5c88c1bR1)

Reply:
There are YAML linters that that have "only when needed" modes for quotes.